snapcraft.yaml

```diff
@@ -382,10 +466,8 @@ parts:
       # Same goes for arm64 builds on Launchpad.  It seems we can't
       # detect $GITHUB_WORKSPACE, so we assume presence of the file
       # "symbols-upload-token" means we are running on GitHub.
-      if [ -f "$CRAFT_PROJECT_DIR/symbols-upload-token"] || [ $CRAFT_ARCH_BUILD_FOR = "arm64" ] || [ $CRAFT_ARCH_BUILD_FOR = "armhf" ]; then
+      if [ $CRAFT_ARCH_BUILD_ON = "armhf" ] || [ $CRAFT_ARCH_BUILD_ON = "arm64" ]; then
         patch -p1 < $CRAFT_PROJECT_DIR/patches/mozilla-reduce-rust-debuginfo.patch
-      fi
-      if [ $CRAFT_ARCH_BUILD_FOR = "armhf" ]; then
```

Comment:
This disables LTO in arm64. Should be a separate commit explaining why.

Reply:
Hm yes this might be a mistake on my side, checking